examples/CURL/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-curl)
+
+hunter_add_package(CURL)
+find_package(CURL REQUIRED)
+find_package(ZLIB REQUIRED)
```

Comment:
Dependency of CURL should be loaded in `CURLConfig.cmake` automatically so users don't have to take care of it manually.